CHANGELOG.md

```diff
@@ -1,5 +1,89 @@
 # Changelog
 
+## Version 0.44.0
+
+([Full Changelog](https://github.com/unitaryfund/mitiq/compare/v0.43.0...v0.44.0))
+
+### Highlights
+
+🚀 This release introduces the first version of the **Virtual Distillation** (VD) technique in Mitiq, which is now available for use!
+This technique was prototyped and implemented by a team of students at the University of Amsterdam.
+VD uses additional qubits to distill a purer version of the quantum state of interest.
+The implementation is in its early stages so lacks support for all `QPROGRAM` types.
+Currently only programs written in `cirq` are suppoted.
+We welcome feedback and suggestions for improvement.
+
+```py
+from mitiq import vd
+
+vd.execute_with_vd(circuit, execute)
+>>> np.array([0.5, 0.5]) # [<Z_0>, <Z_1>] assuming the circuit acts on 2 qubits
+```
+
+We've also made further enhancements to the **Layerwise Richardson Extrapolation** (LRE) technique, including support for observables and the `mitiq.Executor` class.
+`mitiq.lre` also has two new functions `mitiq.lre.construct_circuits` and `mitiq.lre.combine_results` that allow users to generate circuits and combine results in a more modular way (bringing this module in line with the other error mitigation techniques).
+An example workflow for the two step application LRE is shown below:
+
+```py
+from mitiq import lre
+
+lre_circuits = lre.construct_circuits(circuit, degree, fold_multiplier)
+
+results = execute(lre_circuits)
+
+lre_result = lre.combine_results(results, circuit, degree, fold_multiplier)
+```
+
+### Deprecations
+
+For uniformity across modules within Mitiq we have renamed the folowing functions:
+1. `mitiq.zne.scaled_circuits` -> `mitiq.zne.construct_circuits`
```

Comment:
Does `mitiq` give any deprecation warnings?

Reply:
Ah yeah we do not. I guess we should call this section "Breaking changes" or "Backward Incompatible changes"?

Reply:
Breaking changes works